vendor/github.com/ServiceComb/go-archaius/.gitignore

```diff
@@ -0,0 +1,15 @@
+# Binaries for programs and plugins
+*.exe
+*.dll
+*.so
+*.dylib
+
+# Test binary, build with `go test -c`
+*.test
+
+# Output of the go coverage tool, specifically when used with LiteIDE
+*.out
+
+# Project-local glide cache, RE: https://github.com/Masterminds/glide/issues/736
+.glide/
+.idea
```

Comment:
newline please.

Reply:
@m1093782566 the addition of new line also goes back when dep ensure is done. Please confirm whether I should add the same.

Reply:
Okay, no problem.